microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,327 @@ If you publish a person's Personal Information to an outside service or third-pa
+
+This includes content that is:
+* Discriminatory, hateful, or offensive.
+* Harmful or dangerous.
+* Misleading or fraudulent.
+* Spam or malware.
 
-The following policies apply to content and metadata (including publisher name, extension name, extension icon, extension description, extension screenshots, extension trailers and trailer thumbnails, and any other extension metadata) offered for distribution in Microsoft Edge Add-ons.  _Content_ means the images, sounds, videos and text contained in your extension, the tiles, notifications, error messages or ads exposed through your extension, and anything delivered from a server or to which your extension connects.
+Examples of specific types of content that would be considered inappropriate for Microsoft Edge Add-ons include:
+* Pornography or nudity.
+* Hate speech or symbols.
+* Violence or gore.
+* Drugs or alcohol.
+* Gambling.
+* Illegal activities.
+* Misleading or fraudulent claims.
+* Spam or malware.
 
-Because extensions and Microsoft Edge Add-ons are used around the world, these requirements are interpreted and applied in the context of regional and cultural norms.
+The above list of inappropriate content is not exhaustive.  The Microsoft Edge Add-ons store may reject or remove any extension that they deem to be inappropriate, even if it does not violate any specific policy.
 
-### 2.1 Content Requirements for Microsoft Edge Add-on Catalog Listing
+An extension that doesn't meet the requirements for Microsoft Edge Add-ons store listings may be rejected or promptly removed.
 
-Metadata and other content you submit to accompany your extension may not contain mature content.  Submissions that don't meet Microsoft Edge Add-ons store listings requirements are rejected or promptly removed.
 
-### 2.2 Content Including Names, Logos, Original, and Third-party
+<!-- ------------------------------ -->
+#### 2.2 Content including names, logos, and original and third-party content
 
-All content in your extension and associated metadata must be either originally created by the you or appropriately licensed from a third-party rights holder and must be used only as permitted by the rights holder or as otherwise permitted by law.
+All content in your extension and associated metadata must be originally created by you or appropriately licensed from a rights holder and must be used only as permitted by the rights holder or as otherwise permitted by law.
 
-### 2.3 Risk of Harm
+The content must reference Microsoft trademarks only to the extent necessary and should not modify or alter the trademarks in any way.
 
-#### 2.3.1 Requirements
+If not created or appropriately licensed, a disclaimer should be added stating the same
+
+
+<!-- ------------------------------ -->
+#### 2.3 Risk of harm
+
+<!-- todo: add content between headings if corp-level style guide applies here -->
```

Comment:
3. style guides say need content between headings. but these headings are a numbered outline, so does that apply? what could we put here between headings?

Reply:
it's an outline, keeping as-is